### DISCUSSION TITLE
Help with climate card.

### DISCUSSION THREAD

**Discussion by orestkos** [Clooos/Bubble-Card#956] [Questions about config, custom styles and templates]
I have a few questions:

When the thermostat is in heat mode, the dark font on a light theme is hard to read. Is there a way to change the text color depending on the thermostat's state?

To fit all the necessary information, I had to remove the main icon. However, this card doesn't seem to have a "Tab action on button" feature, which is available in other cards.

Is it possible to change the mdi icon in the sub-button depending on the entity state? I tried different approaches, but couldn't get it to work.
![зображення_viber_2024-11-21_23-21-04-906](https://github.com/user-attachments/assets/97ad8874-8e62-41f0-a956-17ddfe5253ab)

**Comment by ChipSkylark37** [comment]
EDIT: Moved into new discussion: https://github.com/Clooos/Bubble-Card/discussions/984

Looking for something similar / related:

Would it be possible that when the system is in AC / Heat mode (for a Heat Pump system) that it could color the entire card with either a 'blue' or 'orange' color to indicate the mode its in? Idle would be the generic color, and then if its in AC, it switches to color the entire card BLUE, or ORANGE if in heat mode.

I looked through some of the 'style' settings but don't know if this is possible.

**Comment by MrBearPresident** [comment]
@orestkos 

Do you still need help with this? 

Have you checked:
- https://github.com/Clooos/Bubble-Card?tab=readme-ov-file#styling
- https://github.com/Clooos/Bubble-Card?tab=readme-ov-file#templates
- https://github.com/Clooos/Bubble-Card/discussions/categories/share-your-custom-styles-templates-and-dashboards

**Reply by ChipSkylark37** [reply]
In the other thread I linked to, I believe this is mostly working, as it relates to styling. 

There were some other questions with being able to add additional entity states to the same HVAC card, and trying to address and issue where the HVAC is in "heat - idle" state, but the system still believes that's "heating" so it colors it orange.

**Reply by MrBearPresident** [reply]
I'm looking over lots of discussions/questions if they have a solutions.
I'm not seeing here in the discussion thread. 

All of what @orestkos is asking is indeed possible with custom-styling. 

If he has a solution he can post it and mark his own answer as the solution.
